src/tls.c

```diff
@@ -10923,7 +10926,10 @@ int TLSX_EarlyData_Use(WOLFSSL* ssl, word32 maxSz, int is_response)
     }
 
     extension->resp = is_response;
-    extension->val  = maxSz;
+    /* In QUIC, earlydata size is either 0 or 0xffffffff.
+     * Override any size between, possibly left from our intial value */
+    extension->val  = (WOLFSSL_IS_QUIC(ssl) && is_response
+                       && maxSz > 0)? UINT32_MAX : maxSz;
```

Comment:
I am not sure we can guarantee `UINT32_MAX` is available? We do have some `UINT_MAX` in our sp_int.h. Not sure what is best here. Also please add space after `?`

Reply:
Can't rely o UINT32_MAX.
We define a WOLFSSL_MAX_16BIT in types.h.
We could add WOLFSSL_MAX_32BIT as well.